packages/components/src/snackbar/list.js

```diff
@@ -3,19 +3,51 @@
  */
 import classnames from 'classnames';
 import { omit, noop } from 'lodash';
-import { useTransition, animated } from 'react-spring/web.cjs';
+import { motion, AnimatePresence } from 'framer-motion';
 
 /**
  * WordPress dependencies
  */
 import { useReducedMotion } from '@wordpress/compose';
-import { useState } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Snackbar from './';
 
+const SNACKBAR_VARIANTS = {
+	init: {
+		height: 0,
+		opacity: 0,
+	},
+	open: {
+		height: 'auto',
+		opacity: 1,
+		transition: {
+			height: { stiffness: 1000, velocity: -100 },
+		},
+	},
+	exit: {
+		x: -300,
+		opacity: 0,
+		transition: {
+			x: { stiffness: 1000, velocity: -100 },
+		},
+	},
+};
+
+const SNACKBAR_REDUCE_MOTION_VARIANTS = {
```

Comment:
This is still an animation, I think in "reduce motion", we disable animations entirely in all Gutenberg animations, so I feel we should just do that here as well. 

Reply:
Good catch 👍